client/src/components/signUpButton/SignUpButton.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const SignUpButton = () => {
+  return <StyledSignUpButton>Registrieren</StyledSignUpButton>;
+};
+
+export default SignUpButton;
+
+const StyledSignUpButton = styled.button`
+  text-align: center;
+  border-radius: 1.8em;
+  padding: 0.6em 1.8em;
+  outline: none;
+  border: none;
+  cursor: pointer;
+  color: var(--font-color-light);
```

Comment:
In this line you use, global variables, in the next you don't- Maybe you could add one here, too? 💯